tests/library/client-certificates.spec.ts

```diff
@@ -25,6 +25,7 @@ const { createHttpsServer, createHttp2Server } = require('../../packages/playwri
 type TestOptions = {
   startCCServer(options?: {
     http2?: boolean;
+    enableHTTP1FallbackWhenUsingHttp2?: boolean;
```

Comment:
drive-by: more strict, enable it only in the test when we need it, line 308. This also makes the new test safer.